doc/source/management/utility_user.rst

```diff
@@ -0,0 +1,106 @@
+.. _psaas_utility_user:
+
+==============
+ Utility user
+==============
+
+In verion 8.0.17-8, |Percona Server| has implemented ability to have a |MySQL| user who has system access to do administrative tasks but limited access to user schema. This feature is especially useful to those operating |MySQL| As A Service. 
+
+This user has a mixed and special scope of abilities and protection:
+
+  * Utility user will not appear in the mysql.user table and can not be modified by any other user, including root.
+
+  * Utility user will increase the result in the `SHOW STATUS LIKE '%Connections'` statement.
```

Comment:
It's either "results" or you could specify that it's about `Threads_connected` and `Threads_created`. And also the `Connections` counter.